Documentation/guides/Extending LiveCode.md

```diff
@@ -614,6 +614,56 @@ support/icon.png // 20x40 png image
 support/icon@extra-high.png // 40x80 png image
 ```
 
+## The Extension Store
+
+Once you have created and packaged your extension you can make it available to the LiveCode Community via the [Extension Store](https://livecode.com/products/extensions), in the [widgets](https://livecode.com/products/widgets/) section.
+
+### Registering as an Extension Developer
+
+The first step is to [register](https://livecode.com/account/developer/register) as an Extension Developer.
+
+- Log in to your LiveCode Account
+- Select the Register option under the Developer section
+
+![enter image description here](images/extensions-developer_id.png)
+
+LiveCode Extensions must have unique identifiers. These identifiers use reverse domain notation.
+
+You can sign up on this page for a LiveCode Community developer ID. Having a developer ID will allow you to use extension identifiers of the form: community.livecode.developerID.extensionName
```

Comment:
I suggest putting the example identifier in `inline code markup` and adding a full stop at the end of the sentence.